src/server/service/kv.rs

```diff
@@ -863,7 +893,8 @@ fn response_batch_commands_request<F>(
     id: u64,
     resp: F,
     tx: Sender<(u64, batch_commands_response::Response)>,
-    timer: HistogramTimer,
+    timer: Instant,
+    label_enum: GrpcTypeKind,
```

Comment:
Can we create a timer helper for local histograms just like the non-local one before?

Reply:
> Can we create a timer helper for local histograms just like the non-local one before?

According to a discussion with @breeswish ,timer for local histograms has some performance issue,so we use Instant here instead.